_posts/2020-11-23-how-fast-are-ractors.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "How Fast are Ractors?"
+date: 2020-11-04 09:00:00
+categories: ["ruby", "performance"]
+author: "noahgibbs"
+---
+
+Ruby 3x3 is coming in about a month. One of its new concurrency primitives is [Ractors](https://github.com/ruby/ruby/blob/master/doc/ractor.md), which used to be called "Guilds." (The other new concurrency primitive is [Autofibers](https://bugs.ruby-lang.org/issues/13618).)
```

Comment:
today matz announced the event, maybe we can link that here?

https://rhc.connpass.com/event/195483

It's in japanese though haha.